2024Robot_Team3484/src/main/include/Constants.h

```diff
@@ -12,6 +26,113 @@ namespace HookConstants {}
 
 
 
-namespace SwerveConstants {}
+namespace SwerveConstants {
+    namespace AutonNames {
+    const std::string AUTON_NONE = "Nothing";
+    const std::string AUTON_DISTANCE = "Drive 5 feet";
+    const std::string AUTON_ANGLE = "Turn 90 degrees";
+    const std::string AUTON_SEQUENCE = "Drive Sequence";
+    }
+
+    namespace ControllerConstants {
+        namespace Driver {
+            constexpr int DRIVER_CONTROLLER_PORT = 0;
+            constexpr double JOYSTICK_DEADBAND = 0.02;
+            #define THROTTLE XBOX_LS_Y 
+            #define STRAFE XBOX_LS_X
+            #define ROTATION XBOX_RS_X
+            #define RESET_HEADING XBOX_BACK
+            #define BRAKE XBOX_X
+            #define STRAIGHTEN_WHEELS XBOX_START
+            #define BRAKE_MODE XBOX_RB
+            #define DISABLE_BRAKE_MODE XBOX_LB
```

Comment:
While I don't think we are at risk of these names duplicating across namespaces, `#define` is not bound by namespace scopes. i.e. if you were to declare `#define BRAKE` under the `Operator` namespace, you would get a compiler error for redeclaring `BRAKE`.

I don't know if there really is an action to be done here, but just something to keep in mind for other constants.